server/src/projects/projects.controller.ts

```diff
@@ -86,15 +91,17 @@ export class ProjectsController {
 
     try {
       if (contactId) {
-        return this.projectsService.getProject(id, contactId);
+        return await this.projectsService.getProject(id, contactId);
```

Comment:
Ditto — why do we need to `return await` ?

Reply:
see comment above